lib/spack/spack/binary_distribution.py

```diff
@@ -520,8 +520,6 @@ def relocate_package(workdir, spec, allow_root):
     old_prefix = str(buildinfo.get('spackprefix',
                                    '/not/in/buildinfo/dictionary'))
     rel = buildinfo.get('relative_rpaths', False)
-    if rel:
-        return
```

Comment:
This is so that text files are still relocated right?  Can we still skip binary reolcation?

Reply:
Yes, we can (and do) still skip binary relocation. There's another conditional later in the function on `rel`.